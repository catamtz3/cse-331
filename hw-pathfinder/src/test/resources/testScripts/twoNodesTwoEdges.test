CreateGraph twoNodes

AddNode twoNodes a
AddNode twoNodes b

AddEdge twoNodes a b 10.0
AddEdge twoNodes b a 5.0

ListNodes twoNodes

ListChildren twoNodes a
ListChildren twoNodes b

FindPath twoNodes a b