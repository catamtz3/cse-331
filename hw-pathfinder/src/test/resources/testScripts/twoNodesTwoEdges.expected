created graph twoNodes

added node a to twoNodes
added node b to twoNodes

added edge 10.0 from a to b to twoNodes
added edge 5.0 from b to a to twoNodes

twoNodes contains: a, b

the children of a in twoNodes are: b(10.0)
the children of b in twoNodes are: a(5.0)

path from A to B:
a to b with weight 10.0
total cost: 10.0